cypress/support/e2e.js

```diff
@@ -152,3 +133,20 @@ export const AddressFormAssertions = {
         cy.contains('[for=f-lookup-postcode] + .govuk-error-message', 'We could not find any addresses for that postcode. Check your postcode is correct, or enter your address manually.');
     }
 }
+
+export const AccessibilityChecks = {
```

Comment:
is this actually used anywhere?

Reply:
Nah, from playing around with Make rules - dropped it just as you commented